Classes/Hooks/ContentPostProcessorHook.php

```diff
@@ -123,7 +123,12 @@ protected function getConfiguration(array $rawConfiguration) {
     foreach ($configuration as $option => $value) {
 
       // Perform stdWrap processing on all options
-      if (StringUtility::isLastPartOfString($option, '.') && isset($value['stdWrap.'])) {
+      if (version_compare(TYPO3_version, '7.0', '>=')) {
```

Comment:
I mean something like `$typo3SevenOrNewer = version_compare(TYPO3_version, '7.0', '>=');` and then `if ($typo3SevenOrNewer) { ... }`. One could even set up a closure depending on the TYPO3 version and fully skip the check within the loop.